client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -0,0 +1,199 @@
+ * style dependencies
+ */
+import './style.scss';
+
+const MarketplaceThankYouContainer = styled.div< MarketplaceThemeProps >`
+	background-color: #fff;
+	overflow: scroll;
+	-ms-overflow-style: none;
+	scrollbar-width: none;
+	&::-webkit-scrollbar {
+		display: none;
+	}
+	margin-top: var( --masterbar-height );
+	height: calc( 100vh - var( --masterbar-height ) );
+`;
+
+const MarketplaceThankYouHeader = styled.div`
+    width: 100%;
+    height: 240px;
+    background-color: var( --studio-gray-0 );
+    display: flex;
+    justify-content: center;
+}`;
+
+const MarketplaceThankyouSection = styled.div`
+    margin-bottom: 35px;
+}`;
+
+const ThankYouBody = styled.div`
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	margin-top: 50px;
+	> div {
+		width: 460px;
+		padding: 0 35px;
+	}
+	div {
+		min-width: 144px;
+	}
+`;
+
+const MarketplaceNextSteps = styled.div< MarketplaceThemeProps >`
+	h3 {
+		font-weight: ${ ( { theme } ) => theme?.weights.bold };
+	}
+
+	p {
+		color: var( --studio-gray-50 );
+		padding-right: 20px;
+	}
+	> div {
+		display: flex;
+	}
+`;
+
+const MarketplaceThankYou = () => {
+	const selectedSite = useSelector( getSelectedSite );
+	const previousPath = useSelector( getPreviousPath );
+	const menuItems = useSiteMenuItems();
+
+	const { url: postsPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'edit-php' ) ?? {};
+
+	const { url: yoastSeoPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'wpseo_dashboard' ) ?? {};
+
+	const translate = useTranslate();
+	const imageWidth = isDesktop() ? 200 : 150;
+
+	return (
+		<>
+			<Masterbar>
+				<Item
+					icon="cross"
+					onClick={ () =>
+						previousPath
+							? page( previousPath )
+							: page(
+									`/plugins/wordpress-seo${ selectedSite?.slug ? `/${ selectedSite.slug }` : '' }`
+							  )
+					}
+					tooltip={ translate( 'Go to plugin' ) }
+					tipTarget="close"
+				/>
+			</Masterbar>
+			<MarketplaceThankYouContainer className="marketplace-thank-you__container">
+				<MarketplaceThankYouHeader>
+					<img alt="yoast logo" width={ imageWidth } src={ yoastInstalledImage }></img>
+				</MarketplaceThankYouHeader>
+				<ThankYouBody>
+					<div>
+						<MarketplaceThankyouSection>
+							<MarketplaceHeaderTitle className="marketplace-thank-you__body-header wp-brand-font">
+								{ translate( 'Yoast SEO Premium is installed' ) }
+							</MarketplaceHeaderTitle>
+						</MarketplaceThankyouSection>
+						<MarketplaceThankyouSection>
+							<MarketplaceHeaderTitle
+								subtitle
+								className="marketplace-thank-you__body-header wp-brand-font"
+							>
+								{ translate( 'What’s next?' ) }
+							</MarketplaceHeaderTitle>
+							<MarketplaceNextSteps>
+								<h3>{ translate( 'Plugin setup' ) }</h3>
+								<div>
+									<p>
+										{ translate(
+											'Get to know Yoast SEO and customize it, so you can hit the ground running.'
+										) }
+									</p>
+									<div>
```

Comment:
It looks like this div only contains 1 element. Could we remove it?

Reply:
This is required since I am flexbox with the div to achieve this. If removed the reusable button component styles seem to be interfering with the layout.